pkg/csi_driver/node.go

```diff
@@ -276,6 +294,9 @@ func (s *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolu
 
 	if mounted {
 		// Already mounted
```

Comment:
log here that staging path already mounted

Reply:
done